apps/api/e2e/utils/test-helpers.ts

```diff
@@ -176,3 +192,26 @@ export function generateRandomString(length: number): string {
   }
   return result;
 }
+
+/**
+ * Generate random hex string of specific length
+ */
+export function generateRandomEthAddress(): string {
```

Comment:
Had to move this here since the `ApiClient` class is no longer being passed into the above helper function.
In the PR to fully migrate to the SDK, we can go back to using a client class instance if desired.